js/script.js

```diff
@@ -6,10 +6,128 @@ const navSlide = () => {
+const popupLive = 'file:///D:/micro_projects/portfolio_mobile/index.html';
+
+const popupSource = 'https://github.com/ZuhRa-HashiMi';
+
+const popupObj = {};
+for (let i = 0; i < popupTitle.length; i += 1) {
+  popupObj[`Project${i}`] = {};
+  popupObj[`Project${i}`].title = popupTitle[i];
+  popupObj[`Project${i}`].technologies = popupTechno;
+  popupObj[`Project${i}`].description = popupDescrip;
+  popupObj[`Project${i}`].liveLink = popupLive;
+  popupObj[`Project${i}`].sourceLink = popupSource;
+}
+
+const popupSection = document.createElement('section');
+popupSection.className = 'popup-section';
+document.body.appendChild(popupSection);
+
+const modal = document.createElement('div');
+modal.className = 'modal';
+popupSection.appendChild(modal);
+
+const newCloseImg = document.createElement('img');
+newCloseImg.className = 'new-close-img';
+newCloseImg.src = 'image/x-icon.png';
+newCloseImg.alt = 'Close icon';
+modal.appendChild(newCloseImg);
+
+const newTitle = document.createElement('h3');
+newTitle.className = 'new-title';
+modal.appendChild(newTitle);
+
+const newTechno = document.createElement('div');
+newTechno.classList.add('new-techno', 'tags');
+modal.appendChild(newTechno);
+for (let i = 0; i < popupTechno.length; i += 1) {
+  const newTechnoItem = document.createElement('h6');
+  newTechnoItem.textContent = popupObj.Project0.technologies[i];
+  newTechno.appendChild(newTechnoItem);
+}
+
+const modalContainer = document.createElement('div');
+modalContainer.className = 'sec-container';
+modal.appendChild(modalContainer);
+
+const newProjectImg = document.createElement('img');
+newProjectImg.className = 'new-project-img';
+newProjectImg.src = 'image/snapshot.jpg';
+newProjectImg.alt = 'project image';
+modalContainer.appendChild(newProjectImg);
+
+const newDescription = document.createElement('p');
+newDescription.className = 'new-description';
+newDescription.textContent = popupObj.Project0.description;
+modalContainer.appendChild(newDescription);
+
+const newBtns = document.createElement('div');
+newBtns.className = 'new-btns';
+modalContainer.appendChild(newBtns);
+const firstBtn = document.createElement('a');
+firstBtn.href = popupObj.Project0.liveLink;
+firstBtn.className = 'button';
+firstBtn.textContent = 'See Live';
+newBtns.appendChild(firstBtn);
+const secondBtn = document.createElement('a');
+secondBtn.href = popupObj.Project0.sourceLink;
+secondBtn.className = 'button';
+secondBtn.textContent = 'See Source';
+newBtns.appendChild(secondBtn);
```

Comment:
- Your project card buttons are missing some icons as seen in the Figma template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏.

Your project 👇:
<img width="288" alt="projectbtn" src="https://user-images.githubusercontent.com/73607512/195582078-c3151e2e-76ec-4a5d-990d-2667f6997a04.PNG">

Figma template:
<img width="135" alt="projectbtn figma" src="https://user-images.githubusercontent.com/73607512/195582165-928d6609-222d-4952-a64e-a1c66e4bc2c4.PNG">